modules/db/src/main/scala/gem/dao/DatasetDao.scala

```diff
@@ -7,30 +7,48 @@ import doobie.imports._
 import java.time.Instant
 
 object DatasetDao {
```

Comment:
I think as we build it out DAOs will do more operations that are compositions of multiple statements (we see a bit of this already) so I'd prefer to keep it as-is for now, and treat `Statements` modules as an implementation detail. Could change my mind but I think it will end up making sense.